language/move-lang/src/parser/syntax.rs

```diff
@@ -577,33 +581,36 @@ fn parse_term<'input>(tokens: &mut Lexer<'input>) -> Result<Exp, Error> {
         }
 
         Tok::NameValue => {
+            let name_loc = make_loc(tokens.file_name(), start_loc, start_loc);
```

Comment:
Sorry that I didn't notice this earlier, but this uses the wrong ending offset. I think you'll need to wait until after the calls to parse_name() where this variable is used, and then you can use tokens.previous_end_loc() to get the right ending offset.

Reply:
Done (after push).